inginious/frontend/plugins/statistics/pages/user_statistics.py

```diff
@@ -1,8 +1,20 @@
+import datetime
 from inginious.frontend.pages.utils import INGIniousAuthPage
+from inginious.frontend.plugins.analytics.manager import AnalyticsManager
 from .constants import base_renderer_path, get_use_minified
 
 class UserStatisticsPage(INGIniousAuthPage):
+
     def GET_AUTH(self, course_id):
+        analyticsmg = AnalyticsManager(self.database)
+        analytics_params = {
+            'username': self.user_manager.session_username(),
+            'service': 'statistics',
+            'date' : datetime.datetime.now(),
```

Comment:
I think the date shouldn't be sent when the visit is generated, by default, the now value can be taken inside `add_visit`, as all of them take the `now()` value.